src/Interop/InteropBinding.cs

```diff
@@ -0,0 +1,157 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Application.Exceptions;
+using Concordium.Sdk.Types;
+
+
+namespace Concordium.Sdk.Interop;
+
+/// <summary>
+/// Contains FFI bindings which can be compiled to work on platforms Linux, OSX and Windows.
+/// </summary>
+[SuppressMessage("Globalization", "CA2101:Specify marshaling for P/Invoke string arguments")]
+internal static class InteropBinding
+{
+    private const string DllName = "librust_bindings";
+
+    [DllImport(DllName, CallingConvention = CallingConvention.Cdecl, EntryPoint = "schema_display")]
+    private static extern bool SchemaDisplay(string schema, FfiByteOption schema_version, ref IntPtr result);
+
+    [DllImport(DllName, CallingConvention = CallingConvention.Cdecl, EntryPoint = "get_receive_contract_parameter")]
+    private static extern bool GetReceiveContractParameter(string schema, FfiByteOption schema_version, string contract_name, string entrypoint, string value, ref IntPtr result);
+
+    [DllImport(DllName, CallingConvention = CallingConvention.Cdecl, EntryPoint = "get_event_contract")]
+    private static extern bool GetEventContract(string schema, FfiByteOption schema_version, string contract_name, string value, ref IntPtr result);
+
+    /// <summary>
+    /// Get module schema in a human interpretable form.
+    /// </summary>
+    /// <param name="schema">Module schema in hexadecimal</param>
+    /// <param name="schemaVersion">Optional schema version if present from module</param>
+    /// <returns>Module schema in a human interpretable form</returns>
+    internal static string? SchemaDisplay(string schema, ModuleSchemaVersion? schemaVersion)
```

Comment:
When is the return value `null`? Please add this to the documentation.

Reply:
Changed return type to not be null